source/common/redis/command_splitter_impl.cc

```diff
@@ -147,9 +147,12 @@ InstanceImpl::InstanceImpl(ConnPool::InstancePtr&& conn_pool, Stats::Scope& scop
       stats_{ALL_COMMAND_SPLITTER_STATS(POOL_COUNTER_PREFIX(scope, stat_prefix + "splitter."))} {
   // TODO(mattklein123) PERF: Make this a trie (like in header_map_impl).
   addHandler(scope, stat_prefix, "expire", all_to_one_handler_);
+  addHandler(scope, stat_prefix, "get", all_to_one_handler_);
```

Comment:
Can we take the opportunity to do what we talked about in person? (Basically just have a list of strings of the all to one handler methods. There are going to be a ton of these). We can then use that list in the tests also.